webpack/move_to_pf/LoadingState/LoadingState.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from 'patternfly-react';
+import './LoadingState.scss';
+
+const LoadingState = ({
+  loading,
+  loadingText,
+  children,
+}) => {
+  if (loading) {
+    return (
+      <div className="loading-state">
+        <Spinner loading={loading} size="lg" />
+        <p>{loadingText}</p>
+      </div>
+    );
+  }
+
+  return children;
+};
+
+LoadingState.propTypes = {
+  loading: PropTypes.bool,
+  loadingText: PropTypes.string,
+  children: PropTypes.node,
+};
+
+LoadingState.defaultProps = {
+  loading: false,
+  loadingText: 'Loading',
```

Comment:
Shouldn't it be translated?

Reply:
I don't believe patternfly-react has any method of translation.

Reply:
I still have a concern that a consumer might use this component with its untranslated default value.
IMHO as long as this component is in katello code base, it should be translated, or alternatively be removed

Reply:
That's fine, updated.